packages/app-store/routing-forms/zod.ts

```diff
@@ -9,6 +9,15 @@ export const zodNonRouterField = z.object({
   selectText: z.string().optional(),
   required: z.boolean().optional(),
   deleted: z.boolean().optional(),
+  options: z
+    .array(
+      z.object({
+        placeholder: z.string(),
```

Comment:
Why are we making placeholder required?

Reply:
Infact we don't need to save placeholder. It is just a UI thing.

Placeholder is only needed when option isn't filled and you can't save without filling the option.